# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../linux24/99-mkdep-arglist-too-long.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Depending on the working directory where the kernel is build, the file
list can grow to produce an "argument to long" error on mkdep line.
This patch only slits the process to have shorter lines.

	- Alejandro

--- ./Makefile.orig	2003-09-23 22:19:55.000000000 -0400
+++ ./Makefile	2003-09-23 22:28:29.000000000 -0400
@@ -518,7 +518,9 @@
 ifdef CONFIG_MODVERSIONS
 	$(MAKE) update-modverfile
 endif
-	scripts/mkdep -- `find $(FINDHPATH) \( -name SCCS -o -name .svn \) -prune -o -follow -name \*.h ! -name modversions.h -print` > .hdepend
+	for x in $(FINDHPATH); do \
+		scripts/mkdep -- `find $$x \( -name SCCS -o -name .svn \) -prune -o -follow -name \*.h ! -name modversions.h -print` >> .hdepend ; \
+	done
 	scripts/mkdep -- init/*.c > .depend
 
 ifdef CONFIG_MODVERSIONS
